### PR TITLE
Text & Icon has been changed as mentioned in MifosX-2323

### PR DESCRIPTION
**Pull request by bnasim**
Provisioning criteria text has been modified and pinterest icon replaced.

### CODE REVIEW

File: app/views/administration/organization.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h4 class="list-group-item-heading"><i class="icon-calendar icon-large"></i>&nbs
                     translate}}</p>
             </a>
             <a class="list-group-item" href="#/viewpaymenttype" has-permission='READ_PAYMENTTYPE'>
-                <h4 class="list-group-item-heading"><i class="icon-pinterest icon-large"></i>&nbsp;&nbsp;
+                <h4 class="list-group-item-heading"><i class="icon-dollar icon-large"></i>&nbsp;&nbsp;
```
Comment (nazeer1100126) [inline app/views/administration/organization.html]: MIFOSX-2323 description is about changing the icon, text for Provisioning. Why do we need to change icon for PAYMENTTYPE?

Comment (bnasim) [inline app/views/administration/organization.html]: I discussed with Ed & Gaurav before changing the icon for payment type.

**Comment by nazeer1100126** [comment]
We are not aware of this discussion :-). At least you should have updated the MIFOSX-2323 description. Can you please reopen the pull request.

**Comment by bnasim** [comment]
This is my first PR. I didn't have any idea who is in charge of merge. pinterest was not appropriate for payment type.

**Comment by nazeer1100126** [comment]
No issues Nasim. I have re-opened your pull request.
Will merge the code after discussing with QA team ;-) 
Thanks

**Comment by bnasim** [comment]
Thank you Nazeer. It's lot of work to open PR repeatedly from my side.

**Pull request merged**